presto-main/src/main/java/io/prestosql/execution/TaskState.java

```diff
@@ -64,4 +70,9 @@ public boolean isDone()
     {
         return doneState;
     }
+
+    public boolean isRunningOrFlushing()
```

Comment:
I'm not a fan of this method.  In all uses, it seem to be used in an `||` with some other clause on the state.  Instead, I would just inline this method.